schemas/predefined_plans.go

```diff
@@ -0,0 +1,70 @@
+package schemas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Peripli/service-manager/pkg/types"
+	"github.com/gofrs/uuid"
+	"time"
+)
+
+const ReferencePlan = `{
+ "name": "reference-instance",
+  "catalog_name": "reference-instance",
```

Comment:
I see many times the same names and strings (for the description for example)
I think it will be better to define them once under a variable so if we have changes we will be aligned for each.

That way we avoid mismatches.

Like we used to have:
`referencePlan.CatalogName = instance_sharing.ReferencePlanName`
`referencePlan.Name = instance_sharing.ReferencePlanName`
`referencePlan.Description = instance_sharing.ReferencePlanDescription`

Reply:
name, catalog_name, displayName etc...
description, translations.lang.description etc...

Reply:
what do you think? 
we might consider using the JSON as template and format it..

Reply:
consider maybe use a `BuildReferenceSchema` function instead of a constant so you will be able to format the schema with the additional constants of the name and description.

Reply:
`func BuildReferencePlanSchema() string {
	return fmt.Sprintf(`{
  "name": "%[1]s",
  "catalog_name": "%[1]s",
  "description": "%[2]s",
  "bindable": true,
  "ready": true,
  "metadata": {
    "supportedPlatforms": [],
    "translations": {
      "en-US": {
        "displayName": "%[1]s",
        "description": "%[2]s"
      }
    }
  },
  "schemas": {
    "service_instance": {
      "create": {
        "parameters": {
          "$schema": "http://json-schema.org/draft-04/schema#",
          "type": "object",
          "additionalProperties": false,
          "_show_form_view": true,
          "properties": {
            "%[3]s": {
              "title": "Referenced Instance ID",
              "description": %[2]s",
              "_title": "TITLE_XTIT",
              "_description": "DESCRIPTION_XMSG",
              "type": "string",
              "minLength": 1,
              "maxLength": 100
            }
          }
        }
      }
    }
  }
}`, instance_sharing.ReferencePlanName, instance_sharing.ReferencePlanDescription, instance_sharing.ReferencedInstanceIDKey)
}`

Reply:
that way we also use the `instance_sharing.ReferencedInstanceIDKey`